src/main/java/vg/civcraft/mc/civmodcore/dao/ManagedDatasource.java

```diff
@@ -62,7 +66,8 @@
 	private static final long MAX_WAIT_FOR_LOCK = 600000l;
 	private static final long WAIT_PERIOD = 500l;
 	
-	private ConnectionPool connections;
+	private HikariDataSource connections;
+	private HikariConfig config;
```

Comment:
Do we really want to retain the HikariConfig after the DataSource has been created? All of the configuration should also be in the Bukkit config, right?